README.md

```diff
@@ -106,7 +106,7 @@ job.run       # => raises AASM::InvalidTransition
 ```
 
 If you don't like exceptions and prefer a simple `true` or `false` as response, tell
-AASM not to be *whiny*:
+AASM not to not be *whiny*:
```

Comment:
This would be a double negative, which reads the same as telling it to be whiny. It should either be `not to` (as it is already) or `to not`: either works in English.